SemanticFormsSelect.php

```diff
@@ -18,6 +18,16 @@
 
 SemanticFormsSelect::initExtension();
 
+$GLOBALS['wgExtensionCredits']['semantic'][] = array(
+	'path' => __FILE__,
```

Comment:
This part went missing with https://github.com/SemanticMediaWiki/SemanticFormsSelect/commit/50f6890a23165990935067c39edd6e2efe2c0b5f.